_pages/en/mandate.md

```diff
@@ -16,14 +16,13 @@ The [Canadian Experience](https://github.com/sara-sabr/ITStrategy/blob/master/wi
 
 ## Mission (team's reason for existence)
 
-Be the catalysts of change within IITB to lay the path for the evolution of ESDC's digital services.
+Enable the strategic value of IT within ESDC by reducing its risks to accelerate business flexibility.
 
 ## Mandate (team's authority within IITB)
 
-- Diagnose key ESDC organisational problems
-- Identify recommended course of action to any ESDC governance body
-- Measure progress of strategies implementation against approved key performance
-- Access and review any ESDC information and documentation as they pertain to process efficiency
+- Diagnose key ESDC organisational problems, waste, lead times and governance structures that inhibits the desired changes in behaviours
+- Recommended specific course of actions to any ESDC governance body to adopt new methods of managing IT risks
+- Measure progress of adoption of strategies implementation in IITB teams and governance bodies as a performance indicator for the IT Strategy team
```

Comment:
This is the only place we mention IITB.  Should we change it just IT?

Reply:
for this particular one, it's best we keep it scoped for IITB. We want to measure teams adoption of strategies as a means to measure our own performance. If we have to deal with outside of IITB, we're going outside our organization.